db/migrate/20151231160304_create_organizations.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOrganizations < ActiveRecord::Migration
+  def change
+    create_table :organizations do |t|
+      t.string :display_name
+      t.string :sector
+      t.string :industry
+      t.string :referral_source
+      t.float :woman_ownership_percent
+      t.float :poc_ownership_percent
+      t.integer :environmental_impact_score
```

Comment:
As we discussed today, these demographic fields will be in their own table, so we should probably leave them out here.